CMakeLists.txt

```diff
@@ -16,9 +16,16 @@ set(cdat_CMAKE_SOURCE_DIR ${cdat_SOURCE_DIR}/CMake)
 set(cdat_CMAKE_BINARY_DIR ${cdat_BINARY_DIR}/CMake)
 set(cdat_external_patch_dir ${cdat_SOURCE_DIR}/exsrc)
 
-# Helper variables to locate programs and libraries
-set(SB_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib")
-set(SB_EXTERNALS_DIR "${CMAKE_INSTALL_PREFIX}/Externals")
+if (APPLE)
+  set(SB_EXTERNALS_DIR "${CMAKE_INSTALL_PREFIX}/Externals")
+  set(SB_DIR "${CMAKE_INSTALL_PREFIX}/Library/Frameworks/Python.framework/Versions/2.7")
```

Comment:
PYTHON_VERSION isn't set at this point; it gets set below, during `include(python_pkg)`. Also, it's set to the entire version of python (2.7.10) and the name of this directory is 2.7; also also, this is the same path we were using already, just relocated and refactored. My CMake-fu is too weak to figure out how to get just 2.7 out of 2.7.10 (oh, and this stuff has to be before `include(scipy_pkg)`, because otherwise if there's a system cython it tries to use that). Any ideas?